deploy/releases/1.2.0/kubectl/readme.adoc

```diff
@@ -58,18 +58,13 @@ curl -L https://raw.githubusercontent.com/OpenLiberty/open-liberty-operator/main
 +
 [source,sh]
 ----
-curl -L https://raw.githubusercontent.com/OpenLiberty/open-liberty-operator/main/internal/deploy/1.2.0/kubectl/openliberty-app-operator.yaml \
+curl -L https://raw.githubusercontent.com/OpenLiberty/open-liberty-operator/main/deploy/releases/1.2.0/kubectl/openliberty-app-operator.yaml \
       | sed -e "s/OPEN_LIBERTY_WATCH_NAMESPACE/${WATCH_NAMESPACE}/" \
       | kubectl apply -n ${OPERATOR_NAMESPACE} -f -
 ----
 
 == Uninstallation
 
-_Optional_: Remove any `OpenLibertyApplication` Custom Resource instances before uninstalling the operator.
-
 To uninstall the operator, run commands from Step 2c first and then Step 2b (if applicable), but after replacing `kubectl apply` with `kubectl delete`.
 
-To delete the CRD, run command from Step 1, but after replacing `kubectl apply` with `kubectl delete`.
-
-_Deleting the CRD will also delete all `OpenLibertyApplication` in the cluster_
-
+Optionally you can delete the CRD resources, but note that deleting the CRD also deletes all instances of the `OpenLibertyApplication`, `OpenLibertyTrace` and `OpenLibertyDump` custom resources in the cluster. Skip this step if you are planning to install the Open Liberty Operator again and want the existing instances of these custom resources to be managed by the new instance of the Operator. To delete the CRD, run command from Step 1, but after replacing `kubectl create` with `kubectl delete`.
```

Comment:
If the CRDs already exist, `kubectl create` will throw an error. Should we recommend them to replace `kubectl create` with `kubectl apply --server-side`?

Reply:
Good point. Updated instances of `kubectl create` in docs with `kubectl apply --server-side`. Thanks